.gitignore

```diff
@@ -75,6 +75,7 @@ compile_commands.json
 # ref: https://github.com/rapidsai/cuml/pull/6201
 pytest.ini
 !python/cuml/cuml/benchmark/automated/pytest.ini
+!python/cuml/cuml_accel_tests/upstream/umap/pytest.ini
 
 # testing
-ci/**/scikit-learn/
```

Comment:
This was unused, leftover from the previous iteration of running sklearn tests in ci.